README.prod.md

```diff
@@ -284,6 +284,24 @@ using topics:
 * You can edit topics to fix a thread that's already been started,
   which can be helpful when onboarding new batches of users to the platform.
 
+Third, setting default streams for new users is a great way to get
+new users involved in conversations before they've accustomed
+themselves with joining streams on their own. You can use the
+[`set_default_streams`](https://github.com/zulip/zulip/blob/master/zerver/management/commands/set_default_streams.py) command to set default streams for users within a realm:
+
+```
+python manage.py set_default_streams --domain=zulip.com --streams=foo,bar,...
+```
+
+And you can use the [`add_users_to_streams`](https://github.com/zulip/zulip/blob/master/zerver/management/commands/add_users_to_streams.py) command to subscribe users to streams:
+
+```
+python manage.py add_users_to_streams --domain=zulip.com --streams=foo,bar,... --all-users
+```
+
+This comes in handy if you change the default streams and some users
+are not yet subscribed to the new defaults.
```

Comment:
Hmm I would probably recommend adding users to streams via the "Streams page" UI when they create them, since that's possible today and a better flow; so I think we probably don't need to document this management command here...